frontend/packages/dev-console/integration-tests/support/pages/functions/checkTerminalIcon.ts

```diff
@@ -0,0 +1,16 @@
+import { webTerminalPO } from '../../pageObjects/web-terminal-po';
+
+export const checkTerminalIcon = (tries: number = 10) => {
+  if (tries < 1) {
+    return;
+  }
+  cy.wait(10000); // wait is for body to render after reload
```

Comment:
This means there will always be a 10 second wait, even if the icon exists before the function is first called. I'd rather only wait if the icon is not found.